src/OrdersApi/Builder/Util/ItemListProvider.php

```diff
@@ -41,9 +41,11 @@ public function __construct(
 
     public function getItemList(CurrencyEntity $currency, OrderEntity $order): ItemCollection
     {
+        $taxStatus = $order->getTaxStatus() ?? $order->getPrice()->getTaxStatus();
+
         $items = new ItemCollection();
         $currencyCode = $currency->getIsoCode();
-        $isNet = $order->getTaxStatus() !== CartPrice::TAX_STATE_GROSS; /* @phpstan-ignore-line */
+        $isNet = $taxStatus !== CartPrice::TAX_STATE_GROSS; /* @phpstan-ignore-line */
```

Comment:
what is the PHPStan issue here? there should at least be a comment, explaining, why this could not be fixed

Reply:
I don't know why it was there, I forgot to remove it